data/compatible_products.yml

```diff
@@ -37,6 +37,15 @@ sets:
 
     - title: Jakarta EE 9 Web Profile
       items:
+      - name: AAS
```

Comment:
This name should match the other Apusic product 'Apusic AAS'. There is some deduplication that happens, iirc

Reply:
@autumnfound , OK, make sense. Thanks Martin!